public/javascripts/developer.js

```diff
@@ -28,6 +28,32 @@ function Developer () {
             minZoom: 9
         }).addLayer(L.mapbox.styleLayer(i18next.t('common:map-url-streets')));
 
+        // URLs for downloading citywide data
+        var cityAttributesWithLabelsCSV = `/v2/access/attributesWithLabels?lat1=${data.southwest_boundary.lat}&lng1=${data.southwest_boundary.lng}&lat2=${data.northeast_boundary.lat}&lng2=${data.northeast_boundary.lng}&filetype=csv`;
```

Comment:
Now that we have your other PR merged that does this by default when no params are passed, let's just keep it simple and pass no parameters. This also means that you should be able to just put the URL right into the `attr()` calls below instead of needing a separate section just to define the URLs